numpy/ma/core.py

```diff
@@ -6824,7 +6824,16 @@ def argsort(a, axis=np._NoValue, kind=None, order=None, endwith=True, fill_value
 argsort.__doc__ = MaskedArray.argsort.__doc__
 
 def sort(a, axis=-1, kind=None, order=None, endwith=True, fill_value=None):
-    "Function version of the eponymous method."
+    """
+    Return a sorted copy of the masked array.
+
+    Equivalent to creating a copy of the array
+    and applying MaskedArray sort() method.
```

Comment:
```suggestion
    and applying the MaskedArray ``sort()`` method.
```
I am wondering if "Refer to `MaskedArray.sort` for the full documentation" in some form may be better/necessary, mainly.